src/main.cpp

```diff
@@ -333,6 +333,26 @@ namespace service_ctrl {
     return false;
   }
 }  // namespace service_ctrl
+
+/**
+ * @brief Checks if NVIDIA's GameStream software is running.
+ * @return `true` if GameStream is enabled.
+ */
+bool
+is_gamestream_enabled() {
```

Comment:
Should we put function somewhere under `src/platform/windows`?

Reply:
I've gone back and forth on this one. The code in `src/platform` right now is actually code that implements common interfaces shared between platforms (the functions in `src/platform/common.h`). 

We could add a `src/platform/windows/win32.h` or similar and declare Windows-specific interfaces there then include it everywhere we need it, but we'd still end up with `#ifdef _WIN32` peppered around so it didn't feel like a big win.